boards/sim/sim/sim/src/sim_bringup.c

```diff
@@ -311,5 +316,15 @@ int sim_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SIM_LINUXI2CBUS
+  /* Initialize the i2c master via the Linux i2c bus device */
+
+  i2cbus = linuxi2c_initialize(9);
```

Comment:
This number of i2c bus shouldn't been hardcode. We should add config or a hardcode i2c path by soft link (/dev/i2c-x)

Reply:
Yes. I will expose this as a config value, just hard coded for my testing.